build_catalog.sh

```diff
@@ -34,7 +36,11 @@ if [ $exists -eq 1 ]; then
   tmp_dir=$(mktemp -d -t sa-XXXXXXXXXX)
   pushd $tmp_dir
     docker export tmp_$$ | tar -xf -
-    prev_version=`find . -name *.clusterserviceversion.* | xargs cat - | yq r - metadata.name`
+    curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/$yq_version/yq_linux_amd64 && chmod +x ./yq
+    prev_version=`find . -name *.clusterserviceversion.* | xargs cat - | ./yq r - metadata.name`
```

Comment:
I'm indifferent to using python for this.  I tend to avoid calling an interpreter for just a one liner, and we already have to download off the internet for the opm binary, so it's not clear using python buys us much from the stand point of reducing the needs of the script (still d/ling from internet).  I don't care either way though.

There can be only 1 csv in the bundle.  If there's more than one, opm will barf and refused to create the catalog.  Arguably we could check here for multiple csv files and die before creating the bundle as well.  However, there must be only 1 csv per bundle.

Yes, we need NAME.VERSION as the replaces is done on the name of the CSV, not the version field of the CSV.

Reply:
Just avoiding dependencies, downloads and more writes in the jobs as much as possible, just that... It's not super critical, though :-D

Reply:
I went ahead and used python.  Between adding a d/l dependency or a python dependency, the python one may be less bad.